### PR TITLE
Add author's bio to the end of every post

### PR DESCRIPTION
**Pull request by etagwerker**
Hey @lubc @ABizzinotto, 

This PR solves: https://www.pivotaltracker.com/story/show/172517172

Please check it out.

Thanks!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -39,41 +39,45 @@ defaults:
...
     github: cleivianecosta
+    bio: "A Brazilian Software Engineer passionate about code and technology. Rubik's Cube Fan, LoL player and coffee lover."
   bronzdoc:
     display_name: "Luis Sagastume"
     gravatar: a223b4e7d8ec959c9aa6668cddcfa605
     email: luis@ombulabs.com
     twitter: bronzdoc
     github: bronzdoc
+    bio: "Software Engineer based in Guatemala city. Passionate about open source, learning new technologies and building cool stuff. People haved called him 'Doc' since fourth grade."
   abizzinotto:
```
Comment (lubc) [inline _config.yml:81]: @etagwerker You can add Amanda's bio too

File: _config.yml [hunk 2]
```diff
@@ -39,41 +39,45 @@ defaults:
...
     twitter:
     github: ercohen14
   cleiviane:
-    display_name: Cleiviane Costa
+    display_name: "Cleiviane Costa"
     gravatar: db42db5ca02074d39438c612433f2209
     email: cleiviane@ombulabs.com
     twitter:
     github: cleivianecosta
+    bio: "A Brazilian Software Engineer passionate about code and technology. Rubik's Cube Fan, LoL player and coffee lover."
```
Comment (cleicar) [inline _config.yml:73]: @etagwerker I need to change my bio. Not a LoL player anymore haha

What do you think about:

"A Brazilian Software Engineer passionate about code, entrepreneurship and technology. She likes to think out of the box and bring new ideas for the team."

Comment (etagwerker) [reply _config.yml]: @cleicar Sure, no problem. I just updated it. 😄 

You might want to submit a PR for https://github.com/ombulabs/fastruby.io (I originally grabbed your bio from that page) 😛

**Comment by etagwerker** [comment]
@cleicar @lubc Just addressed your feedback. 👍

File: _config.yml [hunk 3]
```diff
@@ -39,44 +39,51 @@ defaults:
...
     twitter: bronzdoc
     github: bronzdoc
+    bio: "Software Engineer based in Guatemala city. Passionate about open source, learning new technologies and building cool stuff. People haved called him 'Doc' since fourth grade."
   abizzinotto:
     display_name: "Amanda Bizzinotto"
     gravatar: 31e0846e316c552502506d6cc998fbd1
     email: amanda@ombulabs.com
     twitter: AmandaBizzinot2
     github: ABizzinotto
+    bio: "Experienced Agile and Scrum practitioner. She keeps FastRub.io projects moving and makes sure they cross the finish line."
```
Comment (lubc) [inline _config.yml:87]: @etagwerker typo here:

```suggestion
    bio: "Experienced Agile and Scrum practitioner. She keeps FastRuby.io projects moving and makes sure they cross the finish line."
```

**Comment by lubc** [comment]
@ABizzinotto @etagwerker I just deployed this to staging

**Comment by etagwerker** [comment]
@lubc Looks good 👍 

Screen Shot 2020-04-28 at 9 22 05 PM

**Pull request merged**